units/src/amount.rs

```diff
@@ -1381,45 +1381,56 @@ impl SignedAmount {
     /// - `-1` if the amount is negative
     pub fn signum(self) -> i64 { self.0.signum() }
 
+    /// Checks if this [`SignedAmount`] is positive
+    ///
     /// Returns `true` if this [`SignedAmount`] is positive and `false` if
     /// this [`SignedAmount`] is zero or negative.
     pub fn is_positive(self) -> bool { self.0.is_positive() }
 
+    /// Checks if this [`SignedAmount`] is negative
+    ///
     /// Returns `true` if this [`SignedAmount`] is negative and `false` if
     /// this [`SignedAmount`] is zero or positive.
     pub fn is_negative(self) -> bool { self.0.is_negative() }
 
-    /// Get the absolute value of this [`SignedAmount`].
+    /// Returns the absolute value of this [`SignedAmount`].
+    ///
     /// Returns [`None`] if overflow occurred. (`self == MIN`)
```

Comment:
You could add: "Consider using `unsigned_abs` which is often more practical."

Reply:
Added